web/src/main/java/it/pagopa/selfcare/mscore/web/controller/InstitutionController.java

```diff
@@ -340,19 +342,30 @@ public ResponseEntity<InstitutionResponse> updateInstitution(@ApiParam("${swagge
      *
      * @param request OnboardingInstitutionUsersRequest
      * @return no content
-     * * Code: 204, Message: successful operation
+     * * Code: 200, Message: Ok
+     * * Code: 201, Message: Created
      * * Code: 404, Message: Not found, DataType: Problem
      * * Code: 400, Message: Invalid request, DataType: Problem
      */
+    @ApiResponses(value = {
+    	    @ApiResponse(code = 200, message = "Ok"),
```

Comment:
you could add more information about 200...for ex. that already exists an onboarding in status ACTIVE or SUSPENDED with that productId